onnx/checker.cc

```diff
@@ -733,6 +744,10 @@ void check_graph(const GraphProto& graph, const CheckerContext& ctx, const Lexic
       }
     }
 
+    if (check_is_experimental_op(node.op_type())) {
```

Comment:
I think we should check that the domain is the standard domain also. If it is some other domain, we should not treat it as experimental. (I realize that this is a pre-existing issue, but would be good to fix it.)

Reply:
Good point. I have further considered domain. Please note that I also changed the public function `check_is_experimental_op`'s interface from
`bool check_is_experimental_op(std::string node_op_type)`
to
`bool check_is_experimental_op(const NodeProto& node)`. I think right now check_is_experimental_op is only used internally in onnx repo (shape_inference's implementation.cc) and passing the node itself is simpler. Let me know if you have concern with it. Thanks!